manifests/params.pp

```diff
@@ -74,7 +74,11 @@
   $agent                     = true
 
   # Custom puppetmaster
-  $puppetmaster              = $::puppetmaster
+  if $::trusted['authenticated'] == 'local' {
```

Comment:
It appears the tests exploded because it's undefined. I think it needs a check if ```$::trusted``` is defined.